docs/products/postgresql/howto/migrate-using-bucardo.rst

```diff
@@ -0,0 +1,116 @@
+     ``session_replication_role`` parameter. Aiven uses the open source
+     ``aiven_extras`` extension to allow you to run ``superuser``
+     commands as a different user, as direct ``superuser`` access is not
+     provided for security reasons.
+
+#. | Open and edit the ``Bucardo.pm`` file with administrator
+     privileges.
+   | The location of the file can vary according to your operating
+     system, but you might find it in
+     ``/usr/local/share/perl5/5.32/Bucardo.pm`` , for example.
+
+   #. Scroll down until you see a ``disable_triggers`` function, on line
+      `5324 <https://github.com/bucardo/bucardo/blob/1ff4d32d1924f3437af3fbcc1a50c1a5b21d5f5c/Bucardo.pm#L5324>`__
+
+   #. On line `5359 <https://github.com/bucardo/bucardo/blob/1ff4d32d1924f3437af3fbcc1a50c1a5b21d5f5c/Bucardo.pm#L5359>`__, change ``SET session_replication_role = default`` to
+      the following:
+
+      ::
+
+         $dbh->do(q{select aiven_extras.session_replication_role('replica');});
+
+   #. Scroll down to the ``enable_triggers`` function at line `5395 <https://github.com/bucardo/bucardo/blob/1ff4d32d1924f3437af3fbcc1a50c1a5b21d5f5c/Bucardo.pm#L5395>`__
+
+   #. On line 5428, change ``SET session_replication_role = default`` to
+      the following:
+
+      ::
+
+         $dbh->do(q{select aiven_extras.session_replication_role('origin');});
+
+   #. | Save your changes and close the file.
+
+#. | Add your source and destination databases.
+   | For example:
+
+   ::
+
+      bucardo add db srcdb dbhost=0.0.0.0 dbport=5432 dbname=all_your_base dbuser=$DBUSER dbpass=$DBPASS
+
+      bucardo add db destdb dbhost=cg-pg-dev-sandbox.aivencloud.com dbport=21691 dbname=all_your_base dbuser=$DBUSER dbpass=$DBPASS
+
+#. Add the tables that you want to replicate:
+
+   ::
+
+      bucardo add table belong to us herd=$HERD db=srcdb
+
+   .. note:: You can set ``$HERD`` to any name you choose for the herd, which is used to set up the synchronization.
+
+#. Dump and restore the database from your source to your Aiven service:
+
+   ::
+
+      pg_dump --schema-only --no-owner all_your_base > base.sql
+      psql "$AIVEN_DB_URL" < base.sql
+
+   | You can restore the source data or provide only the schema,
+     depending on the size of your current database.
+
+#. Create the ``dbgroup`` for Bucardo:
+
+   ::
+
+      bucardo add dbgroup src_to_dest srcdb:source destdb:target
+      bucardo add sync sync_src_to_dest relgroup=$HERD db=srcdb,destdb
+      (sudo) bucardo start
+      bucardo status sync_src_to_dest
+
+#. | Start Bucardo and run the ``status`` command. When ``Current state`` is ``Good`` , the data is flowing to your
+     Aiven database.
+
+#. | Log in to the Aiven web console and go to the **Current Queries** tab for your service. This shows you that the ``bucardo`` process is inserting data.
+
+#. Once all your data is synchronized, switch the database connection
+   for your applications to the Aiven service URI.
```

Comment:
Question to CG: Should we rephrase as "...switch the database connection
   for your applications to Aiven for PostgreSQL"? The concern is that the service URI includes the service credentials so do we need to be explicit about not using the service URI directly?

Reply:
True, URI is not needed here and maybe more detail creates more confusion. Your suggestion works for me.